frontend/apps/core-app/src/components/RecipientListTable/RecipientListTableContainer.tsx

```diff
@@ -10,11 +10,13 @@ import { RecipientListTableEntry, SortedFilteredTable } from './types';
 
 type Props<T extends Record> = {
   data: FormWithRecord<T>[][];
+  form: FormDefinition;
```

Comment:
I don't think this needs to be passed in. We can calculate the columns from `FormWithRecord`

Reply:
For that state I'd typically replace the table with some form of placeholder text (the subform table does something like this)